compendium/ServiceComponent/include/cppmicroservices/servicecomponent/detail/ComponentInstance.hpp

```diff
@@ -68,10 +68,11 @@ class US_ServiceComponent_EXPORT ComponentInstance
   virtual void Deactivate() = 0;
 
   /**
-   * This method is called by the runtime while the component configuration is active and if
-   * the configuration properties are modified.
+   * This method is called by the runtime while the component configuration is active when
+   * the configuration properties are modified. Returns false if the component instance has not 
+   * provided a Modified method.
    */
-  virtual void Modified() = 0;
```

Comment:
Jeff/Patty: put Modified() back in, and put InvokeModifiedMethod() into impl class (not visible to public)

Reply:
Couldn't move InvokeModifiedMethod to impl class because it has to be called from code that only has base class pointer. Changed name of InvokeModifiedMethod to Modified. Changed signature of Modified method to return bool instead of void.